.jshintrc

```diff
@@ -5,9 +5,9 @@
 "eqnull" : true,
 "indent": 2,
 "undef": true,
-"unused": true,
-"quotmark": "single",
-"maxlen": 90,
+"unused": false,
```

Comment:
I personally prefer to use `unused: 'vars'`, that way jshint does not complain about unused function parameters, but still checks for unused variables.

See [docs](http://jshint.com/docs/options/#unused):
> This can be set to `vars` to only check for variables, not function parameters, or `strict` to check all variables and parameters. The default (`true`) behavior is to allow unused parameters that are followed by a used parameter.